app/main/views/audits.py

```diff
@@ -180,6 +212,48 @@ def acknowledge_audit(audit_id):
     return jsonify(auditEvents=audit_event.serialize()), 200
 
 
+@main.route('/audit-events/<int:audit_id>/acknowledge-including-previous', methods=['POST'])
```

Comment:
> I don't really see the benefits of keeping it generic across audit event types to be honest.

What I'm *trying* to do is keep what we're doing be an API. If we end up special-casing every frontend view with its own special-case, magic API endpoint, are we making an API anymore? No, we're writing an extremely obtuse distributed monolith.

`/services/<service_id>/updates/acknowledge` *itself* wouldn't be ok as not including the specific event id would introduce a race condition - when an admin acknowledges a version they must be acknowledging *that specific version*, not whatever happens to be the latest one at the time the request gets through.

But `/services/<service_id>/updates/acknowledge-including-previous/<event_id>` wouldn't be the craziest thing in the world, and I see how I could implement it with the potential of expansion to other object types.